Code/Framework/AzFramework/Platform/Common/UnixLike/AzFramework/Device/DeviceAttributesCommon_UnixLike.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+ */
+
+#include <sys/utsname.h>
+#include <AzFramework/Device/DeviceAttributes.h>
+
+namespace AzFramework
+{
+    DeviceAttributeDeviceModel::DeviceAttributeDeviceModel()
+    {
+        m_value = 0;
+        utsname systemInfo;
+        if (uname(&systemInfo) != -1)
+        {
+            m_value = systemInfo.machine;
+        }
+    }
+
+    DeviceAttributeRAM::DeviceAttributeRAM()
+    {
+        m_value = 0;

```

Comment:
No need to set this to 0 here, if a default member initializer is used in the class declaration